.env.example

```diff
@@ -23,4 +23,5 @@ ENABLE_TENANT_AND_SCOPE=true
 TENANTS=dev,test
 TENANT_MIDDLEWARE_EXCLUSION_LIST="/health,/assets/favicon.ico,/pkg/frontend.js,/pkg,/pkg/frontend_bg.wasm,/pkg/tailwind.css,/pkg/style.css,/assets,/admin,/"
 SERVICE_PREFIX=""
-SERVICE_NAME="CAC"
\ No newline at end of file
+SERVICE_NAME="CAC"
+FUNCTION_ENV_VARIABLES="SERVICE_NAME"
```

Comment:
This should not be an ENV. Let us hard code it in file, we have too many ENVs today